### PR TITLE
Feat: Instruct users to setup prerequisites before cluster autoscaler addon.

### PR DESCRIPTION
**Pull request by nileshgadgi**
## what
* Add prerequisites instructions before cluster autoscaler addon setup.

### CODE REVIEW

File: addons/cluster-autoscaler/README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ Cluster Autoscaler is a tool that automatically adjusts the size of the Kubernet
 - there are pods that failed to run in the cluster due to insufficient resources.
 - there are nodes in the cluster that have been underutilized for an extended period of time and their pods can be placed on other existing nodes.
 
+## Prerequisites
+Auto-discovery finds ASGs tags as below and automatically manages them based on the min and max size specified in the ASG.
+- Tag the ASGs with keys to match .Values.autoDiscovery.tags, by default: `k8s.io/cluster-autoscaler/enabled` and `k8s.io/cluster-autoscaler/<YOUR CLUSTER NAME>`
```
Comment (h1manshu98) [inline addons/cluster-autoscaler/README.md:9]: please add the `value` part of above **key** tags

**Comment by h1manshu98** [comment]
Please fix the conflicts and also update `addons/cluster-autoscaler/config/cluster_autoscaler.yaml` with default value.yaml comes with helmchart.

File: addons/cluster-autoscaler/README.md (hunk 1 above)
Comment (nileshgadgi) [reply addons/cluster-autoscaler/README.md:9]: I added value for tag key as you mentioned.

**Comment by nileshgadgi** [comment]
Updated default configuration for cluster autoscaler config file as you mentioned @h1manshu98 
`addons/cluster-autoscaler/config/cluster_autoscaler.yaml`

**Review by h1manshu98 — DISMISSED** [review]
LGTM!

**Review by anmolnagpal — APPROVED** [review]

**Pull request merged**